Apps/Medication/src/Startup.cs

```diff
@@ -62,6 +62,7 @@ public void ConfigureServices(IServiceCollection services)
             this.startupConfig.ConfigureAuditServices(services);
             this.startupConfig.ConfigureAuthServicesForJwtBearer(services);
             this.startupConfig.ConfigureAuthorizationServices(services);
+            this.startupConfig.ConfigureDelegateAuthorizationServices(services);
```

Comment:
This may conflict with your other active PR.

Reply:
they may, but I'm happy to resolve the merge conflict.

Reply:
Is there any particular way you would suggest I go about handling this, it feels a little inevitable with this particular set of PRs?

Reply:
I'd suggest just enabling it in the first one that needs it.  The second one can rely on it being already added.